src/documents/activities/FunctionalMashup.md

```diff
@@ -31,8 +37,8 @@ If desired and available you can provide additional tools and materials that can
  **Tip**: In addition to the object cards, you can use any available materials to prototype your idea.
 
 ## Why
+
 The goal of this activity is to promote creative thinking over immediate feasibility. It encourages participants to dig deeper to find unexpected and creative ways of addressing a problem.
-<ul class="idg-articleContentUseWhyHow">
-<li><span role="presentation" class="idg-iconPractices">{{{getCategoryIcon "practices"}}}</span> [Practice Co-Design](/practices/PracticeCoDesign.html)</li>
-<li><span role="presentation" class="idg-iconPractices">{{{getCategoryIcon "practices"}}}</span> [Collaborate](/practices/Collaborate.html)</li>
-</ul>
+
+* {% include 'svg/icon-practices.svg' %} [Practice Co-Design](/practices/PracticeCoDesign.html)
+* {% include 'svg/icon-practices.svg' %} [Collaborate](/practices/Collaborate.html)
```

Comment:
Not sure why, but the default bullets are appearing for this list only.

![image](https://user-images.githubusercontent.com/564762/97348231-421e2480-1864-11eb-8b0c-6f22c084556c.png)

Reply:
Weird! I'll investigate.

Reply:
Resolved in 719f242.